guessingGame.js

```diff
@@ -2,37 +2,69 @@ let newGuessingGame = require('./lib/newGuessingGame');
 let guessingGameMakeGuess = require('./lib/guessingGameMakeGuess');
 let guessingGameIsDone = require('./lib/guessingGameIsDone');
 
-let wordToGuess = 'hello';
+let process = require('process');
 
-let game = newGuessingGame('hello');
+let readlineSync;
+try {
+  readlineSync = require('readline-sync');
+} catch (err) {
+  console.log('You must install the readline-sync module. Run the following');
+  console.log('command to install it:');
+  console.log();
+  console.log('  npm install');
+  console.log();
+
+  process.exit();
+}
+let secretInput = readlineSync.question('Enter a secret word: ', { hideEchoBack: true });
+
+//console.log(`You entered: ${secretInput}`);
+
+//let wordToGuess = readlineSync.question('What word do you want them to guess? ');
 
-console.log(`"Guessing" the word '${wordToGuess}', one letter at a time.`);
+
+let game = newGuessingGame(secretInput);
+
+console.log(`"Guessing" 'secret word', one letter at a time.`);
```

Comment:
This isn't true anymore; the guessing happens interactively, via user input. Definitely remove *this* `console.log` statement.